inc/spbc-settings.php

```diff
@@ -599,12 +599,17 @@ function spbc_settings__register()
                                 'description'      => __('Will search for known malicious signatures in files. Unknown files will be shown in the results only if both options heuristic analysis and signature analysis are enabled.', 'security-malware-firewall'),
                                 'long_description' => true,
                             ),
-                            'scanner__dir_exclusions'                        => array(
+                            'scanner__dir_exclusions_view'                        => array(
                                 'type'        => 'field',
                                 'input_type'  => 'textarea',
                                 'title'       => __('Directory exclusions for the malware scanner:', 'security-malware-firewall'),
                                 'title_first' => true,
-                                'description' => __('Input relative directories (WordPress folder is ROOT). Separate each directory path by a new line.', 'security-malware-firewall'),
+                                'description' => spbc_get_exclusions_stat('dir') ?: sprintf(
+                                    '<p>%s<br>%s<br>%s</p>',
+                                    __('Input relative directories (WordPress folder is ROOT). Separate each directory path by a new line.', 'security-malware-firewall'),
+                                    __('If you want, you can add file with name spbct_directory_exclusions.txt to the list, where each line will be a directory path to exclude.', 'security-malware-firewall'),
+                                    __('Something like this: mysite.org/wp-content/spbct_directory_exclusions.txt', 'security-malware-firewall')
+                                ),
                                 'long_description' => true
```

Comment:
Is long descritpion provided and updated to current state? The same question for domains exclusions.

Reply:
Yep